tests/test_extra.py

```
@@ -37,12 +52,63 @@
+	python tests (as software), are NOT covered by the same Licensed. Assembly may be required.
+"""
+
+if not _has_docs:
+	try:
+		import docs.utils
+		_has_docs = True
+	except ImportError:  # pragma: no branch
+		_has_docs = False
+
+
+def onlyIfHasDocs(has_docs: bool) -> callable:
```

Ruff (0.11.9)

Boolean-typed positional argument in function definition

(FBT001)